wasatch/AndorDevice.py

```diff
@@ -375,12 +418,20 @@ def init_tec_setpoint(self) -> SpectrometerResponse:
         minTemp = c_int()
         maxTemp = c_int()
         assert(self.SUCCESS == self.driver.GetTemperatureRange(byref(minTemp), byref(maxTemp))), "unable to read temperature range"
-        self.detector_temp_min = minTemp.value
-        self.detector_temp_max = maxTemp.value
 
-        self.setpoint_deg_c = self.detector_temp_min
-        #assert(self.SUCCESS == self.driver.SetTemperature(self.setpoint_deg_c)), "unable to set temperature midpoint"
-        log.debug(f"set TEC to {self.setpoint_deg_c} C (range {self.detector_temp_min}, {self.detector_temp_max})")
+        self.settings.eeprom.max_temp_degC = maxTemp.value
+        self.settings.eeprom.min_temp_degC = minTemp.value
+
+        # commenting-out because Andor camera is reporting -120C for a device 
```

Comment:
It's worth double checking me on this, but I believe it reports -120C because the andor camera can have water cooling hooked up and when it is water cooled it can go down to -120C. I'm guessing Andor doesn't provide a distinction for their air cooling.